src/main/resources/db/changesets/20220217_addphsidandconsentcodecolumns.yaml

```diff
@@ -0,0 +1,19 @@
+databaseChangeLog:
+  - changeSet:
+      id: add_phs_id_and_consent_code_columns
+      author: sh
+      changes:
+        - addColumn:
+            tableName: dataset
+            columns:
+              name: phs_id
+              type: varchar(100)
```

Comment:
If these are values with a known length, we can use that length instead of `100`

Reply:
Unfortunately, I'm not sure that they are.  It's a varchar, so I don't think it hurts to have it be longer than needed

Reply:
Ah, darn. Oh well.